csharp/test/Ice/metrics/MetricsAMDI.cs

```diff
@@ -52,12 +31,16 @@ public ValueTask
 
         public ValueTask opByteSAsync(byte[] bs, Current current) => new ValueTask(Task.CompletedTask);
 
-        public IObjectPrx? getAdmin(Current current)
+        public ValueTask<IObjectPrx?> getAdminAsync(Current current)
         {
             TestHelper.Assert(current != null);
```

Comment:
Useless check, current cannot be null.

Reply:
Well that's probably why it `Assert`s it!
Removed regardless.